### DISCUSSION TITLE
Some settings are not applied

### DISCUSSION THREAD

**Discussion by Solarunit** [home-assistant/frontend#15752] [Configuration]
Home Assistant 2023.3.1, docker

Some settings are not applied, like Dark theme, or Always hide sidebar. Well, they would be applied only for a browser session. Is it docker limitation?

![](https://i.imgur.com/VE4yJxk.png)

![](https://i.imgur.com/Y7efdPh.png)

**Comment by ildar170975** [comment]
> applied only for a browser session

This is a correct behaviour.
Imagine that you opened HA on a mobile client and then can select a different theme & hide a sidebar.

**Reply by Solarunit** [reply]
>This is a correct behaviour.

I wish I could force these settings for HA in the browser, it's annoying to set them every time.

>Imagine that you opened HA on a mobile client and then can select a different theme & hide a sidebar.

I have HA client app installed, no problem there, you never logout from it. I haven't tried a different theme, it's Dark.
Maybe the app should have its own settings?

**Reply by ildar170975** [reply]
In my experience, when I set some settings (like a theme, number format etc) on some iOS Companion App (using 2 devices) or on Win-desktop (Chrome) - these settings are memorized and do not reset - unless I reset mobile clients in Companion App or clear browser cache on Chrome.
Also, in Chrome after clearing a browser's cache, I have to login again (and then set a theme, number format, arrange sidebar items) - and if I do not tick a "login automatically" (called like this) checkbox, then when I open a new Chrome tab I have to login again - but my settings are memorized.
So, if you experience a "user settings are not memorized" issue - I suggest you either to ask a question in Community (I would prefer this, may be you will attract more users) or create an issue.

**Reply by Solarunit** [reply]
>Chrome after clearing a browser's cache, I have to login again

Yes, that's expected, you nothing can do with this, except a dedicated desktop app..

>and then set a theme, number format, arrange sidebar items

this is my problem, I want persistent HA settings every time I login in the Browser.

>I suggest you either to ask a question in Community (I would prefer this, may be you will attract more users) or create an issue.

Good idea, maybe I should try this :slightly_smiling_face:
